modules/reference/pages/command-line-tools.adoc

```diff
@@ -695,6 +695,68 @@ Show help message and exit.
 
 
 
+[[spacewalk-repo-sync]]
+== Synchronize yum Repositories into {productname} Channels (spacewalk-repo-sync)
```

Comment:
yum => software

repomd Repositories would be correct, but this tool can also mirror debian repos. So let's just be generic and say "software repositories"

Reply:
great, much better :)